wazuh/wazuh-dashboard-plugins
# Add overview home last 24 hours KPI

### Description
This pull request removes the pending and never connected agents KPIs from the home view and also adds a new last 24 hours alerts KPI.

The last 24 hours alerts KPI is a call to action that redirects to the Discover plugin with the predefined filters.
 
### Issues Resolved
- #6367 

### Evidence
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/9343732/32f1e404-d0e1-450d-8988-17b7f1e61294)


### Test

- Go to Home -> Overview
- Check the only KPIs that exist are: `Active agents`, `Disconnected agents`, `Last 24 hours alerts`
- Check the `Last 24 hours alerts` is also a link to the Discover plugin
  - The Discover plugin must have the `cluster.name` or `manager.name` pre-filtered (whether the cluster mode is enabled or not)
  - The Discover plugin must be filtered by the last 24 hours
  - The Discover plugin must use `wazuh-alerts-*` index pattern
  - The results of the Discover plugin must match the _Last 24 hours alerts_ KPI count number
- All the cases should be tested with different APIs selected. Check the `cluster.name` or manager.name matches the API selected
  

### Check List
